pkg/gui/containers_panel.go

```diff
@@ -496,6 +496,27 @@ func (gui *Gui) handleContainersRemoveMenu(g *gocui.Gui, v *gocui.View) error {
 	return gui.createMenu("", options, len(options), handleMenuPress)
 }
 
+func (gui *Gui) handleContainerPause(g *gocui.Gui, v *gocui.View) error {
+	container, err := gui.getSelectedContainer()
+	if err != nil {
+		return nil
+	}
+
+	return gui.WithWaitingStatus(gui.Tr.PausingStatus, func() error {
+		if container.Details.State.Paused {
+			err = container.Unpause()
+		} else {
+			err = container.Pause()
+		}
+
+		if err != nil {
+			return gui.createErrorPanel(gui.g, err.Error())
+		}
+
+		return gui.refreshContainersAndServices()
```

Comment:
I'd see how it goes without this line here. Lazydocker should automatically recognise that the container is paused and then refresh accordingly

Reply:
It does not.

Reply:
I've done some digging and this looks to be an actual bug in the docker package where we refetch container info after docker tells us there's been a change but the change doesn't come through. Crazy!

Reply:
Alright, now that we have the murderer, what are we going to do with it? The victim is apparently fine with `gui.refreshContainersAndServices()`

Reply:
Yeah let's just stick to calling `refreshContainersAndServices` and we can look into the deeper issue in another PR